Global/JetBrains.gitignore

```diff
@@ -64,3 +74,7 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+# Maven
```

Comment:
Please submit a fresh PR with context into this template if you have Maven-specific changes to propose: [`Maven.gitignore`](https://github.com/github/gitignore/blob/master/Maven.gitignore)